flink-runtime/src/test/java/org/apache/flink/runtime/operators/coordination/OperatorCoordinatorHolderTest.java

```diff
@@ -57,6 +60,12 @@ public class OperatorCoordinatorHolderTest extends TestLogger {
     private final GlobalFailureHandler globalFailureHandler = (t) -> globalFailure = t;
     private Throwable globalFailure;
 
+    private MetricRegistryImpl registry =
```

Comment:
You can simply return an `UnregisteredJobManagerJobMetricGroup` for testing.
`MetricRegistryImpl` is heavy and must be closed after the testing.